docs/contributing/our-meet-ups.md

```diff
@@ -0,0 +1,21 @@
+# How to collaborate with the _earthaccess_ team
+
+## Bi-weekly (alternating weeks) _earthaccess_ hack days
+
+???+ info "How to get invited"
```

Comment:
What is `???+` supposed to indicate? 🤔

Reply:
Confused me at first as well! It's so weird how many specifications there are for callouts/admonitions... 

Short answer: `???` is a collapsible callout, `+` means to default to expanded, `info` is the "style" of the callout, and `"How to get invited"` is a custom title. :sweat_smile: 

Long answer: https://squidfunk.github.io/mkdocs-material/reference/admonitions/ :)